gis/openstreetmap-carto/Portfile

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        gravitystorm openstreetmap-carto 5.3.1 v
+
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+# license CC0-1.0 (http://creativecommons.org/publicdomain/zero/1.0/)
+license             public-domain
+
+description         A general-purpose OpenStreetMap mapnik style, in CartoCSS
+
+long_description    These are the CartoCSS map stylesheets for the Standard \
+                    map layer on OpenStreetMap.org.
+
+checksums           rmd160  b7efb038f0bf942ff8403f72fa34fb2040fa6fdb \
+                    sha256  fd8f398a2a674a84e070e9eacaa4c0cb7de036950f7434e8c259e6b7974b47df \
+                    size    2719400
+
+python.versions     36 37 38 39
```

Comment:
this does not do anything if the portname does not start with `py-`, so you can remove this line. The `python.default_version` one below will set which Python version will be used.

Reply:
Thanks.  Done.